scripts/swift-analyzer.py

```
@@ -0,0 +1,116 @@
+
+        file_type = get_file_type(filename)
+        lines = patch.split('\n')
+        line_number = None
+
+        for line in lines:
+            if line.startswith('@@'):
+                # Parse hunk header: @@ -old +new @@
+                try:
+                    new_info = line.split('+')[1].split(' ')[0]
+                    line_number = int(new_info.split(',')[0]) - 1
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)